source/_posts/2023-07-05-release-20237.markdown

```diff
@@ -0,0 +1,710 @@
+configuration has been imported in previous releases and can now be safely
+removed from your YAML configuration files.
+
+([@gjohansson-ST] - [#94090]) ([documentation](/integrations/soundtouch))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#94090]: https://github.com/home-assistant/core/pull/94090
+
+{% enddetails %}
+
+{% details "Conversation" %}
+
+When using the Assist shortcut on iOS, it will now always talk to Home Assistant
+conversation agent even if the OpenAI or Google LLM conversation integrations
+are set up.
+
+([@balloob] - [#95225]) ([documentation](/integrations/conversation))
+
+[@balloob]: https://github.com/balloob
+[#95225]: https://github.com/home-assistant/core/pull/95225
+
+{% enddetails %}
+
+{% details "ESPHome" %}
+
+As an additional security hardening measure, the default for allowing ESPHome
+devices to make service calls has changed. If you want to permit the ESPHome
+device to make service calls, it must be enabled in the options flow.
+
+- For existing devices, calling Home Assistant services continues to be allowed.
+- For newly configured devices, it must now be enabled in the options flow.
+
+([@bdraco] - [#95143]) ([documentation](/integrations/esphome))
+
+[@bdraco]: https://github.com/bdraco
+[#95143]: https://github.com/home-assistant/core/pull/95143
+
+{% enddetails %}
+
+{% details "Home Assistant Supervisor" %}
+
+The default name given to backups created with the `hassio.full_backup`
+and `hassio.partial_backup` services is now longer `""` but the date and time
+the backup was initiated (`%Y-%m-%d %H:%M:%S`, for example,
+"2023-07-05 13:37:42").
+
+([@ludeeus] - [#94468]) ([documentation](/integrations/hassio))
+
+[@ludeeus]: https://github.com/ludeeus
+[#94468]: https://github.com/home-assistant/core/pull/94468
+
+{% enddetails %}
+
+{% details "MQTT" %}
+
+If the `initial` temperature, `min_temp`, or `max_temp` is not set, the default
+temperature will be converted to the set `temperature_unit` or system default
+temperature unit if this is not degrees Celsius.
+
+([@jbouwh] - [#93965]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#93965]: https://github.com/home-assistant/core/pull/93965
+
+---
+
+The behavior of MQTT climate when `power_command_topic` is set has been changed:
+
+- The `hvac_mode` state attribute is no longer optimistically set if
+  `power_command_topic` is defined and `climate.turn_on` or `climate.turn_off`
+  is called.
+- When `hvac_mode` is changed, `payload_on` or `payload_off` messages will
+  not be published on the `power_command_topic`.
+
+This also means `power_command_topic` is no longer deprecated.
+
+([@jbouwh] - [#94832]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#94832]: https://github.com/home-assistant/core/pull/94832
+
+{% enddetails %}
+
+{% details "MusicCast" %}
+
+If you have any automation depending on the `source` state attribute of the
```

Comment:
I'd start the paragraph by saying what changed. Ie, the state attribute `source` value has changed from eg. `net_radio` to `Net Radio`.